### PR TITLE
allow to scroll horizontally with mouse wheel

### PR DESCRIPTION
**Pull request by wiktor-mazur**
* **Please check if the PR fulfills these requirements**
- [/] The commit message follows our guidelines
- [/] Tests for the changes have been added (for bug fixes / features)
- [/] Docs have been added / updated (for bug fixes / features)
- [/] Unit tests are passing `ng test`
- [/] Lint tests are passing `ng lint`

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)
User can scroll with mouse wheel only vertically

* **What is the new behavior (if this is a feature change)?**
User can now also scroll with mouse wheel horizontally which improves UX especially when scrollbar is hidden.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No, it is backwards compatibile.

* **Other information**:

### CODE REVIEW

File: src/ngx-drag-scroll.ts [hunk 1]
```diff
@@ -685,6 +694,24 @@ export class DragScrollComponent implements OnDestroy, AfterViewInit, OnChanges,
     return scrollLeftPos >= this._contentRef.nativeElement.scrollWidth;
   }
 
+  @HostListener('wheel', ['$event'])
+  private onWheel(event: WheelEvent) {
+    if (this._xWheelEnabled) {
+      event.preventDefault();
+
+      if (this._snapDisabled) {
+          console.log('x');
```
Comment (wiktor-mazur) [inline src/ngx-drag-scroll.ts]: Actually I've just noticed that console.log left behind :/
Will you fix that on the nearest occasion or should I make another PR?

Comment (bfwg) [reply src/ngx-drag-scroll.ts]: Ah, I need to pay more attention to code reviews. I'll get it fixed.

Comment (wiktor-mazur) [reply src/ngx-drag-scroll.ts]: Great, thanks!
